concordium-consensus/src/Concordium/Skov/Update.hs

```diff
@@ -253,8 +254,8 @@ processFinalization newFinBlock finRec@FinalizationRecord{..} = do
 --    it is added to the appropriate pending queue.  'addBlock'
 --    should be called again when the pending criterion is fulfilled.
 -- 3. The block is determined to be valid and added to the tree.
-addBlock :: forall pv m. (HasCallStack, TreeStateMonad pv m, SkovMonad pv m, FinalizationMonad m, OnSkov m) => PendingBlock -> m UpdateResult
-addBlock block = do
+addBlock :: forall pv m. (HasCallStack, TreeStateMonad pv m, SkovMonad pv m, FinalizationMonad m, OnSkov m) => PendingBlock -> Maybe [TV.VerificationResult] -> m UpdateResult
```

Comment:
Please document the additional parameter, why is is there, when is it Nothing, when is it Just.

Reply:
I removed the `Maybe` from [VerificationResult] and now fetching the already computed transaction verification results for the case of executing pending blocks.